File: 0485-max-consecutive-ones/0485-max-consecutive-ones.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        max_count = 0
        count = 0
        for ele in nums:
            if ele == 1:
                count+=1
                max_count= max(max_count,count)
            else:
                count = 0
            
        return max_count

File: 0485-max-consecutive-ones/0485-max-consecutive-ones.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var findMaxConsecutiveOnes = function(nums) {
    
    let count = 0
    let maxi = 0
    
    for (let i = 0;i<nums.length;i++){
        if(nums[i]===1){
            count +=1
            maxi = Math.max(count,maxi) 
        }
        else{
            count = 0
        }
            
    }
    return maxi
    
};